civis/parallel.py

```diff
@@ -409,7 +409,7 @@ class JobSubmissionError(Exception):
     pass
 
 
-def _robust_result_download(output_file_id, client, n_retries=5, delay=0.0):
+def _robust_pickle_download(output_file_id, client, n_retries=5, delay=0.0):
```

Comment:
It's private, which is why I didn't bother before. In the other function, I though it was helpful to say what the kwargs were for. I added a longer docstring for this function.